tensorflow/core/api_def/base_api/api_def_AddN.pbtxt

```diff
@@ -7,4 +7,8 @@ Must all be the same size and shape.
 END
   }
   summary: "Add all input tensors element wise."
+  ```python
+  `x` is an input tensor with value [9, 7, 10]
```

Comment:
Please write examples inside code blocks as code, in this case 

x = [9, 7, 10]